client/src/components/SearchForm.js

```diff
@@ -0,0 +1,16 @@
+const SearchForm = ({handleSubmit, onChange, city}) => {
+    return (  <form onSubmit={handleSubmit}>
+        <input
+          id="city-name"
+          type="text"
+          placeholder="Please enter the city name"
+          name="city"
+          onChange={onChange}
+          value={city}
+          required
+        />
+        <input type="submit" value="Submit" onClick={handleSubmit} />
+      </form> );
+}
+ 
+export default SearchForm;
```

Comment:
If you're going to put the form in its own component, it should probably manage its own state, so I'd move the city state into this component, then call `handleSubmit` with the form values. This would help performance a bit because it won't cause the whole app to rerender every time the user types a character. Only the form would rerender.